### PR TITLE
Replace python with python3 in incremental_load.sh.tpl

### PR DESCRIPTION
**Pull request by dennajort**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
On MacOS 12.3, Apple removed the default `python` executable.
Building images doesn't work and throw this error:
`env: python: No such file or directory`

Issue Number: #2046 2046

## What is the new behavior?
Build on a system without `python` executable (as MacOS 12.3+) works as expected.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by linzhp — COMMENTED** [review]
Can you sign CLA?

**Comment by dennajort** [comment]
I think I did sign it @linzhp

**Review by linzhp — APPROVED** [review]

File: container/incremental_load.sh.tpl [hunk 1]
```diff
@@ -115,7 +115,7 @@ function find_diffbase() {
     NEW_DIFF_IDS+=("${diff_id}")
   done
 
-  TOTAL_DIFF_IDS=($(cat "${name}" | python -mjson.tool | \
+  TOTAL_DIFF_IDS=($(cat "${name}" | python3 -mjson.tool | \
```
Comment (linzhp) [inline container/incremental_load.sh.tpl:118]: For backward-compatibility of old systems that don't have python3, can you do this instead?

```suggestion

  export PYTHON="python"
  if command -v python3 &>/dev/null; then
      PYTHON="python3"
  fi

  TOTAL_DIFF_IDS=($(cat "${name}" | $PYTHON -mjson.tool | \
```

**Review by linzhp — CHANGES_REQUESTED** [review]

File: container/incremental_load.sh.tpl (hunk 1 above)
Comment (dennajort) [reply container/incremental_load.sh.tpl:118]: Sounds good to me !
At first I was thinking if there is a way to pull this info directly from bazel, but I'm not very skilled in it and didn't find a way.

Comment (linzhp) [reply container/incremental_load.sh.tpl]: rules_docker doesn't manage Python toolchain on its own and depends on system Python instead. So this is the only way.

**Review by linzhp — APPROVED** [review]

**Pull request merged**